src/routers/datatypeRouters/nodes/llm_query.ts

```diff
@@ -39,9 +42,18 @@ async function query (input: { query: string, password: string }): Promise<any>
     })
   }
   const jsonObj = await response.json()
+  if (input.verbose === 'true') {
+    return {
+      query: input.query,
+      aql: jsonObj.aql_query,
+      aql_result: jsonObj.aql_result,
```

Comment:
I would limit both aql and the aql to a certain max number of characters. It can get out of control really fast

Reply:
They can take the AQL and run themselves on the dev server too to get all results and paginate.

Reply:
please disregard this, the AQL queries are always limited by 5